emojis/chef.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 98.64"><title>ChefLogoAsset 1</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path d="M98.78,78.18h0l-.87-1.33v1.88h-.35V76.28h.37l.88,1.35.87-1.35H100v2.45h-.34V76.85Zm-2.6.55h-.35V76.61H95v-.33H97v.33h-.79Z" style="fill:#435363"/><path d="M0,87.08V87A11.42,11.42,0,0,1,11.71,75.4a11.59,11.59,0,0,1,8.86,3.47L17.42,82.5A8.29,8.29,0,0,0,11.68,80c-3.79,0-6.51,3.14-6.51,7V87c0,3.85,2.66,7.06,6.51,7.06,2.57,0,4.14-1,5.9-2.63l3.14,3.18a11.48,11.48,0,0,1-9.21,4A11.34,11.34,0,0,1,0,87.08" style="fill:#435363"/><polygon points="26.45 75.79 31.39 75.79 31.39 84.68 40.51 84.68 40.51 75.79 45.45 75.79 45.45 98.25 40.51 98.25 40.51 89.23 31.39 89.23 31.39 98.25 26.45 98.25 26.45 75.79" style="fill:#435363"/><polygon points="51.48 75.79 68.43 75.79 68.43 80.19 56.39 80.19 56.39 84.74 66.98 84.74 66.98 89.14 56.39 89.14 56.39 93.86 68.59 93.86 68.59 98.25 51.48 98.25 51.48 75.79" style="fill:#435363"/><polygon points="75.05 75.79 92.15 75.79 92.15 80.28 79.99 80.28 79.99 85.06 90.7 85.06 90.7 89.56 79.99 89.56 79.99 98.25 75.05 98.25 75.05 75.79" style="fill:#435363"/><path d="M46.63,45.44a14.11,14.11,0,1,1,13.55-18h5.46a19.41,19.41,0,1,0,0,7.83H60.18A14.13,14.13,0,0,1,46.63,45.44" style="fill:#435363"/><path d="M37.11,40.84a13.42,13.42,0,0,0,9.51,3.95V39.16a7.81,7.81,0,0,1-5.54-2.29Z" style="fill:#435363"/><path d="M33.17,31.33a13.37,13.37,0,0,0,1.14,5.42l5.15-2.26a7.83,7.83,0,0,1,7.17-11V17.87A13.47,13.47,0,0,0,33.17,31.33" style="fill:#f38b00"/><path d="M51.51,18.79l-2,5.25a7.86,7.86,0,0,1,3.94,3.38H59.5a13.52,13.52,0,0,0-8-8.63" style="fill:#435363"/><path d="M49.47,38.63l2,5.25a13.52,13.52,0,0,0,8-8.63H53.41a7.86,7.86,0,0,1-3.94,3.38" style="fill:#f38b00"/><path d="M46.63,57.35a25.93,25.93,0,0,1-18.39-7.63l-3.76,3.76A31.33,31.33,0,0,0,77.71,35.25H72.35a26.06,26.06,0,0,1-25.72,22.1" style="fill:#f38b00"/><path d="M46.63,5.31a25.87,25.87,0,0,1,15.55,5.18l3.18-4.27A31.33,31.33,0,0,0,17.42,20l5,1.93A26.06,26.06,0,0,1,46.63,5.31" style="fill:#f38b00"/><path d="M72.35,27.41h5.36a31.14,31.14,0,0,0-4.51-12.67l-4.52,2.82a25.85,25.85,0,0,1,3.66,9.85" style="fill:#435363"/><path d="M66.31,27.41h5.38A25.44,25.44,0,0,0,55.82,7.7L53.9,12.64A20.13,20.13,0,0,1,66.31,27.41" style="fill:#f38b00"/><path d="M46.63,11.27h0V6h0A25.4,25.4,0,0,0,23,22.13l4.94,1.92a20.1,20.1,0,0,1,18.7-12.79" style="fill:#435363"/><path d="M26.56,31.33h-5.3A25.4,25.4,0,0,0,37.21,54.87l2-4.92A20.09,20.09,0,0,1,26.56,31.33" style="fill:#f38b00"/><path d="M46.63,51.39v5.3A25.4,25.4,0,0,0,71.69,35.25H66.31A20.1,20.1,0,0,1,46.63,51.39" style="fill:#435363"/><path d="M17.94,43.94A31.28,31.28,0,0,1,15.3,31.33h5.32a26.09,26.09,0,0,0,2.2,10.47Z" style="fill:#435363"/></g></g></svg>
```

Comment:
SVGs don't seem to be supported?

https://slack.com/intl/en-nl/help/articles/206870177-Add-custom-emoji-to-your-workspace

Reply:
Odd, it's on slackmoji's as an svg

Reply:
Yeah slack 100% doesn't like svg's. Changed it now :+1: